packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -349,6 +361,20 @@ class Slider extends React.Component {
     this.emitChange(event, value);
   };
 
+  endInteraction(event) {
```

Comment:
`endInteraction` is to generic. What about `handleDragEnd` which is more idiomatic react and is composed of a cleanup and call to `onDragend`.